helmfile/values/networkpolicy/service-cluster.yaml.gotmpl

```diff
@@ -22,6 +22,19 @@ global:
   externalLoadBalancer: {{ .Values.networkPolicies.global.externalLoadBalancer }}
   ingressUsingHostNetwork: {{ .Values.ingressNginx.controller.useHostPort }}
 
+  # TODO: Add Swift
```

Comment:
TODO part of another PR?

Reply:
Yeah, it's for the great refactoring of netpols.